spec/requests/shopping_list_items_spec.rb

```diff
@@ -45,17 +45,21 @@
               expect(response.status).to eq 201
             end
 
-            it 'returns all shopping lists for the same game' do
+            it 'returns all changed shopping lists for the same game' do
               create_item
-              expect(response.body).to eq(game.shopping_lists.reload.index_order.to_json)
+              expect(response.body).to eq(game.shopping_lists.to_json)
```

Comment:
In this case it's the same - these are all the shopping lists. The `reload` and `index_order` are redundant.